udata_front/views/gouvfr.py

```diff
@@ -111,17 +111,40 @@ def get_object(model, id_or_slug):
     return obj
 
 
+def get_objects_from_tag(model, tag) -> list:
+    objects = getattr(model, "objects").filter(tags=tag)
+    return [r for r in objects]
```

Comment:
I think  you could directly return `objects`
```suggestion
    return objects
```